nvflare/private/fed/utils/fed_utils.py

```diff
@@ -93,3 +95,18 @@ def listen_command(listen_port, engine, execute_func, logger):
             conn.close()
         if listener:
             listener.close()
+
+
+def check_client_replies(replies, client_sites: List[str], command: str):
+    display_sites = ",".join(client_sites)
+    if not replies:
+        raise RuntimeError(f"Failed to {command} to the clients {display_sites}: no replies.")
+    if len(replies) != len(client_sites):
+        raise RuntimeError(f"Failed to {command} to the clients {display_sites}: not enough replies.")
```

Comment:
does replies has site information ? if yes, it might be better to tell which site did not get reply